php.nix

```diff
@@ -26,12 +23,26 @@ let
 
       # This derivation uses static links to reduce the build output's size
       inherit (pkgs.pkgsStatic) stdenv lib fetchurl;
-      inherit (pkgs.pkgsStatic) pkgconfig libxml2 bzip2 oniguruma openssl;
-      inherit (pkgs) zlib;
+      inherit (pkgs.pkgsStatic) pkgconfig libxml2 zlib;
+
+      # Static oniguruma (this is pending an upstream merge in nixpkgs)
```

Comment:
Is there a specific location we could include a link to so that this upstream dependency is more clearly indicated?

Reply:
Yes - NixOS/nixpkgs#75950 and/or NixOS/nixpkgs#76659, whichever gets merged first (the latter is a generalization of the former). I'll add them to the comment.